modules/hibernate-4.2/src/test/java/org/apache/ignite/cache/hibernate/HibernateL2CacheTransactionalSelfTest.java

```diff
@@ -75,7 +76,7 @@
 
     /** {@inheritDoc} */
     @Override protected void beforeTestsStarted() throws Exception {
-        jotm = new Jotm(true, false);
+        jotm = new Jotm(true, false, new RmiLocalConfiguration());
```

Comment:
What does it mean and why is it needed?

Reply:
2-arg constructor in new Jotm version will call 3-arg constructor with null value for RMI configuration, but null value for RMI configuration will lead to NPE in other Jotm methods when you run tests.